src/modules/core/components/IndexModal/IndexModalItem.tsx

```diff
@@ -42,7 +50,7 @@ const ColonyActionsDialogItem = ({
           <FormattedMessage {...description} />
```

Comment:
So I think the description doesn't have the correct color:

![Screenshot from 2020-11-30 19-48-25](https://user-images.githubusercontent.com/1193222/100645231-0daaf600-3345-11eb-84ac-8e656ef6c163.png)

And in the Figma file it looks like this:

![Screenshot from 2020-11-30 19-48-29](https://user-images.githubusercontent.com/1193222/100645254-16033100-3345-11eb-8eec-05faa6629d24.png)